src/Utils.php

```diff
@@ -60,6 +60,42 @@ public static function sanitizeShopDomain(string $shop, ?string $myshopifyDomain
         }
     }
 
+    /**
+     * Builds query strings that are compatible with Shopify's format for array handling
+     * Example: IDs = [1,2,3]
+     * PHP would generate:  ids[]=1&IDs[]=2&IDs[]=3
+     * Shopify expects:     ids=["1","2","3"] (URL encoded)
```

Comment:
What would this look like for associative arrays?

Reply:
I was wondering the same thing! I even thought about various ways of implementing those, but

1) I could not find a standards/RFC document that clearly outlined how that was _supposed_ to look - only found a couple of resources demonstrating how some languages/frameworks handle that.
2) I was not sure whether we need this.
3) I was not sure what our server side implementation of that looked like.

TL;DR: Too much uncertainty and no proven use-case at hand. Was not enough to keep me from shipping this in time for our next release.